api/app/tests/weather_models/test_machine_learning.py

```diff
@@ -41,33 +43,26 @@ def given_an_instance(coordinate: List, points: List):
         target_coordinate=coordinate,
         station_code=None,
         max_learn_date=datetime.now())
-# pylint: disable=redefined-outer-name, unused-argument
 
 
-@ when('The machine learns')
-def learn(given_an_instance: machine_learning.StationMachineLearning,
-          mock_get_actuals_left_outer_join_with_predictions: List):
+@when('The machine learns')
+def learn(instance: machine_learning.StationMachineLearning):
     """ Train the machine learning model """
-    given_an_instance.learn()
-# pylint: enable=redefined-outer-name
+    instance.learn()
 
 
-# pylint: disable=redefined-outer-name
 @ then('The <model_temp> for <timestamp> results in <bias_adjusted_temp>')
 def assert_temperature(
-        given_an_instance: machine_learning.StationMachineLearning,
+        instance: machine_learning.StationMachineLearning,
         model_temp: float, timestamp: datetime, bias_adjusted_temp: float):
     """ Assert that the ML algorithm predicts the temperature correctly """
-    result = given_an_instance.predict_temperature(model_temp, timestamp)
+    result = instance.predict_temperature(model_temp, timestamp)
     assert result == bias_adjusted_temp
-# pylint: enable=redefined-outer-name
 
 
-# pylint: disable=redefined-outer-name
 @ then('The <model_rh> for <timestamp> results in <bias_adjusted_rh>')
```

Comment:
since you deleted the spaces for @when, etc, just pointing out that the 'then's still have spaces

Reply:
Thanks - sometimes vscode and I fight.

Reply:
haha oops. Just realized I tagged someone.